metrics/lib/json.bash

```diff
@@ -93,7 +93,8 @@ EOF
 		metrics_json_add_fragment "$json"
 	else
 		if [ "$RUNTIME" == "runc" ]; then
-			local output=$(docker-runc -v)
+			local get_binary=$(sudo docker info --format "{{json .Runtimes.runc}}" | cut -d '"' -f4)
```

Comment:
try with `.Runtimes.runc.Path`, maybe `cut` is not needed

Reply:
@devimc thanks, however, I think I need the cut to remove the ""

Reply:
@devimc the output that I get is "docker-runc" or "runc", however, I still need to remove the "

Reply:
then remove `json`

Reply:
sudo docker info --format "{{.Runtimes.runc}}"

Reply:
@devimc thanks it worked :)